src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ManifestTest.cs

```diff
@@ -119,6 +119,7 @@ public void Bug12935 ([Values (true, false)] bool useAapt2)
 				Assert.IsFalse (builder.Build (proj), "Build for TargetFrameworkVersion 15 should have failed");
 				StringAssertEx.Contains (useAapt2 ? "APT2259: " : "APT1134: ", builder.LastBuildOutput);
 				StringAssertEx.Contains (useAapt2 ? "APT2067" : "", builder.LastBuildOutput);
+				StringAssertEx.Contains ("Properties/AndroidManifest.xml(6)", builder.LastBuildOutput);
```

Comment:
Looks like this new assert is failing: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3690462&view=ms.vss-test-web.build-test-results-tab&runId=12949274&resultId=100141&paneView=debug

```
String did not contain 'Properties/AndroidManifest.xml(6)'!
```

Also, might this assert be macOS-specific?

Reply:
Its is Os specific, a Path separator :'( 

`Properties\AndroidManifest.xml(6)` vs `Properties/AndroidManifest.xml(6)`

🤦